test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

```diff
@@ -1,65 +1,71 @@
 const { expect } = require('chai');
 const { uniqWith } = require('lodash');
-const { isFieldDuplicated } = require('../../utils/utils');
+const { isFieldDuplicated, loadAllFiles } = require('../../utils/utils');
 const { createAssertExists } = require('../../utils/assertBuilders');
 
-const load = require;
-
-const coreEvents = load('definitions/divorce/json/CaseEvent/CaseEvent.json');
-
-const authCaseEventCommon = Object.assign(require('definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent'), {});
-
 const assertEventExists = createAssertExists('Event');
 
-function mergeJsonNonProdFiles() {
-  const definitions = []
-    .concat(load('definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-deemed-and-dispensed-nonprod.json'))
-    .concat(load('definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json'));
-
-  return [...authCaseEventCommon, ...definitions];
-}
-
-function mergeJsonProdFiles() {
-  const definitions = []
-    .concat(load('definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-prod.json'));
-
-  return [...authCaseEventCommon, ...definitions];
-}
+const getAuthorisationCaseEventDefinitions = loadAllFiles('AuthorisationCaseEvent');
+const getCaseEventDefinitions = loadAllFiles('CaseEvent');
```

Comment:
Nice 👍. By the way, would you know is javascript could allow to do something like: Get 'CaseEvent-*-nonprod.json'? That would make things easier as we add files in the future.

Reply:
@Dizrupto This basically sets up the `CaseEvent` it does not automatically load all the documents. You have to specify which one you want to load, examples below